src/store/auth/reducer.ts

```diff
@@ -67,6 +68,7 @@ const authReducer = (
       return {
         ...state,
         checkingInstance: false,
+        isAuthenticated: true,
```

Comment:
need to check and will separate to new action (under store/storage/reducer ; username)